convert_hf_to_gguf.py

```diff
@@ -7730,6 +7957,16 @@ def parse_args() -> argparse.Namespace:
         "--mmproj", action="store_true",
         help="(Experimental) Export multimodal projector (mmproj) for vision models. This will only work on some vision models. A prefix 'mmproj-' will be added to the output file name.",
     )
+    parser.add_argument(
+        "--mistral-format", action="store_true",
+        help="Whether the model is stored following the Mistral format.",
+    )
+    parser.add_argument(
+        "--n-ctx",
```

Comment:
Could we maybe have a list of predefined mapping `model --> n_ctx` instead?

Having this as a required param is quite confusing tbh. Ideally it should be inside `params.json`, otherwise we should make it such that users are not required to enter it manually.